internal/pkg/cli/dependencies/repository_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build !windows
+// +build !windows
+
```

Comment:
Na Windows občas padají tyto testy (https://github.com/keboola/keboola-as-code/runs/6657337483?check_suite_focus=true). Nejspíš to bude stejný problém jako s `git` packagem, tak bych je disabloval taky. 🙂

Reply:
ok